Clear/main.py

```diff
@@ -0,0 +1,337 @@
+#! /usr/bin/python -u
+# date: 07/12/17
+
+import click
+import os
+import subprocess
+from click_default_group import DefaultGroup
+
+try:
+    # noinspection PyPep8Naming
```

Comment:
These `# noinspection` markups are specific to the PyCharm IDE. Please remove them.